src/components/SurveyAnswer/ChoiceAnswer/MultiplePickAnswer.tsx

```diff
@@ -0,0 +1,74 @@
+import Image from 'next/image';
+import SimpleBar from 'simplebar-react';
+
+import { SurveyAnswerProps } from '..';
+
+import styles from './ChoiceAnswer.module.scss';
+import 'simplebar-react/dist/simplebar.min.css';
+
+const MultiplePickAnswer = ({
+  question,
+  currentResponse,
+  onResponseChange,
+}: SurveyAnswerProps) => {
+  const onAnswerSelect = (answerId: string) => {
+    const currentAnswers = currentResponse?.answers || [];
+    const isAnswerSelected = currentAnswers.some(({ id }) => id === answerId);
```

Comment:
The names here seems to conflict somehow... 🤔 
=> It took me time to understand the meaning of the code as I though `selected` and `current` were different concepts. But it turns out they are the same idea... 

I would suggest either:
```suggestion
    const currentAnswers = currentResponse?.answers || [];
    const isCurrentAnswer = currentAnswers.some(({ id }) => id === answerId);
```
(an answer is selected if it is one of the currentAnswers)

or 

```suggestion
    const selectedAnswers = selectedResponse?.answers || [];
    const isAnswerSelected = selectedAnswers.some(({ id }) => id === answerId);
```

Or shifting everything to `selected`.

Reply:
Updated in [d80b942](https://github.com/hanam1ni/react-surveyor/pull/97/commits/d80b942a0e32bf194dea9f5fe6ebeb172a7b95be)